packages/engine/src/simulation/command.rs

```diff
@@ -33,23 +43,25 @@ struct RemoveCommand {
     uuid: Uuid,
 }
 
-/// TODO: DOC
```

Comment:
Maybe something like this?
> Collection of queued commands for creating and deletion of agents.

Reply:
Sure, doesn't hurt. I just found the struct to be pretty self-explanatory